### PR TITLE
SSO-Okta Doc Updates

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

File: v6/enterprise/sso/admin_sso.md [hunk 1]
```diff
@@ -14,30 +14,48 @@ Go to [Team Settings](https://app.getpostman.com/dashboard/teams/edit) for an en
...
 To configure a new authentication method, click **Add a new authentication method** button.
 
 In the **ADD AUTHENTICATION METHOD** modal, select the authentication type. Enter an authentication name that is easily identifiable to your team. Then click the **Proceed** button.
 
 [![authentication method](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/ENT-add-authentication-method-2.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/ENT-add-authentication-method-2.png)
 
-**Note**: Always check with your authentication provider dashboard or your IT support staff for the correct information to complete a modal. 
+**Note**: Always check with your authentication provider dashboard or your IT support staff for the correct information to complete a modal.
 
-In the "Service Provider Details (Postman)", enter the ID for the Entity, and the URLs for the Login and ACS.
```
Comment (aamir-ahmed) [inline v6/enterprise/sso/admin_sso.md]: Please blur the identity provider details.

Comment (ArifPMan) [reply v6/enterprise/sso/admin_sso.md]: resolved.

**Comment by ArifPMan** [comment]
@aamir-ahmed Your f/b is incorporated. You may review the changes.

File: v6/enterprise/sso/admin_sso.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Go to [Team Settings](https://app.getpostman.com/dashboard/teams/edit) for an en
 
 [![sso enable](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/SSO_AdminEnable1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/SSO_AdminEnable1.png)
 
-After configuring an SSO authentication for your Postman team, you can use the toggle option in Settings (as illustrated above) to turn on/off this SSO authentication. Click the orange circled button to turn on/off your SSO authentication system. This is a team-level option which will enable/disable SSO for the whole team.
+After configuring an SSO authentication for your Postman team, you can use the toggle option in Settings (as illustrated above) to turn on/off this SSO authentication. Click the button (circled above) to turn on/off your SSO authentication system. This is a team-level option which will enable/disable SSO for the whole team.
 
 To configure a new authentication method, click **Add a new authentication method** button.
```
Comment (aamir-ahmed) [inline v6/enterprise/sso/admin_sso.md]: @ArifPMan, Before you would see this screen, you would find the attached General Settings screen when you navigate to Team Settings, over here you would have to click on Authentication tab on the left side navbar to get to the image you are referring.
Team Settings-Screen

File: v6/enterprise/sso/admin_sso.md (hunk 1 above)
Comment (aamir-ahmed) [reply v6/enterprise/sso/admin_sso.md]: Please blur the service provider details seen in the attached image. 
Configure Identity Provider details screen

**Comment by ArifPMan** [comment]
@Andy-GetPostman Would you be able to review and approve these changes?

**Pull request merged**